_includes/header.html

```diff
@@ -35,7 +35,7 @@
         <ul class="navigation-bar navigation-bar-right">
           <li class="dropdown">
             <a class="dropdown-toggle" id="docs-page" href="/documentation">DOCS</a>
-            <ul class="dropdown-menu">
+            <ul class="dropdown-menu docs-menu">
               <li><a href="/documentation/#sec0">INTRODUCTION</a></li>
```

Comment:
@yashwin please use descriptive menu like
instead of `/documentation/#sec0` use `/documentation/#Introduction`